certificate-manager/src/main/java/io/strimzi/certs/OpenSslCertManager.java

```diff
@@ -372,18 +381,61 @@ public void addCertToTrustStore(File certFile, String certAlias, File trustStore
     }
 
     @Override
-    public void addKeyAndCertToKeyStore(File keyFile, File certFile, String alias, File keyStoreFile, String keyStorePassword) throws IOException {
-        new OpensslArgs("openssl", "pkcs12")
-                .opt("-export")
-                .optArg("-in", certFile)
-                .optArg("-inkey", keyFile)
-                .optArg("-name", alias)
-                .optArg("-out", keyStoreFile)
-                .optArg("-passout", "pass:" + keyStorePassword)
-                .optArg("-certpbe", "aes-128-cbc")
-                .optArg("-keypbe", "aes-128-cbc")
-                .optArg("-macalg", "sha256")
-                .exec();
+    public void addKeyAndCertToKeyStore(File keyFile, File certFile, String alias, File keyStoreFile, String keyStorePassword)
```

Comment:
Just curious...Shouldn't this method be moved to the `CertManager` class instead of being in `OpenSslCertManager` as it does not use any OpenSSL invocation method?